opaque_keys/edx/keys.py

```diff
@@ -89,6 +89,37 @@ def make_asset_key(self, asset_type: str, path: str) -> AssetKey:  # pragma: no
         raise NotImplementedError()
 
 
+class LibraryCollectionKey(OpaqueKey):
+    """
+    An :class:`opaque_keys.OpaqueKey` identifying a particular Library Collection object.
+    """
+    KEY_TYPE = 'collection_key'
+    __slots__ = ()
+
+    @property
+    @abstractmethod
+    def org(self) -> str | None:  # pragma: no cover
+        """
+        The organization that this collection belongs to.
+        """
+        raise NotImplementedError()
+
+    @property
+    @abstractmethod
+    def lib(self) -> str | None:  # pragma: no cover
+        """
+        The lib for this collection.
+        """
+        raise NotImplementedError()
+
+    @property
+    def library_key(self) -> LearningContextKey:  # pragma: no cover
```

Comment:
I can't use `LibraryLocatorV2` because of a circular import

Reply:
Can you put it in quotes? https://peps.python.org/pep-0484/#forward-references

Reply:
The quotes don't work, but I found this [solution](https://github.com/openedx/opaque-keys/pull/335/commits/168c373f4adafe7c81f707baef3a971f7404acd7#diff-6117aaf0e3d3aaf8569961ad62a10f24b38af67ea69267477abacaf7f071676dR15)